lib/websockets/spaceEvents.ts

```
@@ -42,6 +45,73 @@ export class SpaceEventHandler {
         log.error('Error subscribing user to space events', { error });
         this.sendError('Error subscribing to space');
       }
+    } else if (message.type === 'page_deleted') {
+      try {
+        const pageWithSpaceId = await prisma.page.findUnique({
+          where: {
+            id: message.payload.pageId
+          },
+          select: {
+            spaceId: true,
+            parentId: true
+          }
+        });
+
+        if (!pageWithSpaceId) {
```

should just throw an error here